crates/diffbot_lib/src/github/graphql.rs

```diff
@@ -109,31 +114,28 @@ pub async fn get_pull_files<I: Into<InstallationId>>(
     let mut ret = vec![];
 
     loop {
-        let queried: QueryData = crab
-            .graphql(&format!(
-                "
-query {{
-  repository(owner:\"{}\", name:\"{}\") {{
-    pullRequest(number:{}) {{
-      files(first:100, after:\"{}\") {{
-        edges {{
-          cursor
-          node {{
-            path
-            changeType
-          }}
+        let queried: QueryReturn = crab
+            .graphql(&Query {
+                query: format!(
+                    "{{ repository(owner: \"{}\", name: \"{}\")
```

Comment:
Is there any equivalent to like, prepared statements, or some other kind of proper escaping instead of just putting data in a format string?

Reply:
There is, but this is sufficient, as we're not doing anything that complex here

Reply:
I was just thinking about injection attacks, is it always safe to take something from a github api request and put it back into graphql?

Reply:
As far as I can tell this isn't a realistic concern because GitHub seems to sanitize usernames and repos, probably not worth worrying about. The two 'user' inputs here are given by secured message from github with hmac verification, so it'd only happen if github fucked up.